eva-accession-release/src/main/resources/create_release_properties.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+if (( $# != 8 )); then
+	echo  "This script creates a '.properties' file for the eva-accession-release pipeline, taking the properties from the import, and suggesting commands to run all the release for an assembly.
+This script needs the next parameters in order:
+- The path to the import properties file with information about the assembly to release
+- The output folder for the release, where this script will create a folder for the assembly
+- The path to the eva-accession-release java jar
+- The path to the bgzip executable
+- The path to the tabix executable
+- The path to the script count_ids_in_vcf.sh
+- The path to the vcf-validator executable
+- The path to the vcf-assembly-checker executable
+"
+
+else
+	IMPORT_FILE=$1
+	OUTPUT_RELEASE_FOLDER=$2
+	EVA_ACCESSION_JAR_PATH=$3
+	BGZIP_PATH=$4
+	TABIX_PATH=$5
+	COUNT_IDS_PATH=$6
+	VCF_VALIDATOR_PATH=$7
+	VCF_ASM_CHECKER_PATH=$8
+
+	if [ ! -f ${IMPORT_FILE} ]; then
+		echo "The properties file ${IMPORT_FILE} doesn't exist! Stopping script"
+		exit 1
+	fi
+
+	OUTPUT_FILE=application.properties
+	if [ -f ${OUTPUT_FILE} ]; then
+		mv ${OUTPUT_FILE} ${OUTPUT_FILE}.bk
+		echo "# Made a backup of \"${OUTPUT_FILE}\" into \"${OUTPUT_FILE}.bk\"";
+		if [ ${OUTPUT_FILE} = ${IMPORT_FILE} ]; then
+			IMPORT_FILE=${IMPORT_FILE}.bk
+		fi
+	fi
+
+	# extract values from the existing properties file
```

Comment:
Except for assemblyAccession values for other variables don't seem to be used elsewhere except for the new properties file. So why not copy the lines verbatim for others instead of using "sed"?